crossopt/assignment3/unittests.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+import unittest
+from assignment3.dictionary import Dictionary
+from assignment3.wordsearch import Grid
+from assignment3.wordsearch import findMatch, findMatches
+
+
+class TestDictionary(unittest.TestCase):
+    def test_isprefix(self):
+        dictionary = Dictionary(['CAR', 'CART', 'CARD', 'CAT'])
+        self.assertEqual(dictionary.isPrefix('A'), 0)
```

Comment:
I am not sure I understand well: isPrefix returns a boolean and the unit test compares it to 0 or 1. 
Can you confirm the test pass?

Reply:
I've fixed it, however the tests pass on my computer.